core/templates/pages/preferences-page/preferences-page.component.html

```diff
@@ -34,12 +34,16 @@ <h2 class="oppia-preferences-page-heading-subtext">
         </label>
         <div class="col-lg-8 col-md-8 col-sm-8">
           <div class="oppia-editable-section oppia-editor-profile-picture-container"
+               tabindex="0"
                title="{{ 'I18N_PREFERENCES_CHANGE_PICTURE' | translate }}">
             <div class="oppia-click-to-start-editing e2e-test-photo-clickable"
                  (click)="showEditProfilePictureModal()"
                  aria-label="Edit profile picture button">
             </div>
-            <i class="fas fa-pen oppia-editor-edit-icon oppia-editor-edit-icon-pos"></i>
+            <i class="fas fa-pen oppia-editor-edit-icon oppia-editor-edit-icon-pos"
+               tabindex="0"
+               (keydown.enter)="showEditProfilePictureModal()">
```

Comment:
This should be (click) i think?

Reply:
@heyimShivam Click event will not trigger the enter key....That's why (keydown.enter) is used here .....